### PR TITLE
Add new reference to OpenShift Upgrade Graph

### PR DESCRIPTION
**Pull request by codyhoag**
cc @vikram-redhat

### CODE REVIEW

**Comment by codyhoag** [comment]
This has an exception to merge as soon as the proper acks are given.

File: modules/update-mirror-repository.adoc [hunk 1]
```diff
@@ -9,7 +9,7 @@ Before you upgrade a cluster on infrastructure that you provision in a restricte
 
 .Procedure
 
-. Review the link:https://access.redhat.com/solutions/4583231[{product-title} upgrade paths] to confirm that there is an upgrade path between your current and intended cluster versions.
+. Review the link:https://access.redhat.com/solutions/4583231[{product-title} upgrade paths] to confirm that there is an upgrade path between your current and intended cluster versions. You can use the link:https://access.redhat.com/labs/ocpupgradegraph/update_channel[OpenShift Upgrade Graph visualizer and update planner] to view channel graphs or plan an upgrade from one version to another.
```
Comment (sferich888) [inline modules/update-mirror-repository.adoc:12]: This kcs already points to this tool, so its a bit redundant to list both of these. 
I would remove the KCS and just point to the new tool.

Comment (codyhoag) [reply modules/update-mirror-repository.adoc]: Updated :+1:

**Review by vikram-redhat — APPROVED** [review]
lgtm for merging after peer review.

File: modules/update-mirror-repository.adoc [hunk 2]
```diff
@@ -9,7 +9,7 @@ Before you upgrade a cluster on infrastructure that you provision in a restricte
 
 .Procedure
 
-. Review the link:https://access.redhat.com/solutions/4583231[{product-title} upgrade paths] to confirm that there is an upgrade path between your current and intended cluster versions.
+. Use the link:https://access.redhat.com/labs/ocpupgradegraph/update_channel[OpenShift Upgrade Graph visualizer and update planner] to plan an upgrade from one version to another. The OpenShift Upgrade Graph provides channel graphs and a way to confirm that there is an update path between your current and intended cluster versions
```
Comment (bergerhoffer) [inline modules/update-mirror-repository.adoc:12]: Missing a period

**Review by bergerhoffer — COMMENTED** [review]
Just a missing period, otherwise LGTM!

File: modules/update-mirror-repository.adoc (hunk 2 above)
Comment (bergerhoffer) [reply modules/update-mirror-repository.adoc:12]: Actually, should the title of it be "Red Hat OpenShift Container Platform Update Graph" like is shown when you look at the app?

Comment (codyhoag) [reply modules/update-mirror-repository.adoc]: Good point; it probably should match the app exactly. I just pushed that update :+1:

**Review by ghost — APPROVED** [review]
this docs addition for the OpenShift Upgrade Graph Labs application looks good to me!

**Pull request merged**